suspicious-funding-ts/src/agent.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  Finding,
+  HandleAlert,
+  AlertEvent,
+  TransactionEvent,
+  LabelQueryOptions,
+  ethers,
+  getEthersProvider,
+} from "forta-agent";
+import {
+  BOTS_TO_MONITOR,
+  DAYS_TO_LOOK_BACK,
+  VALUE_THRESHOLDS,
+  alertOriginMap,
+} from "./constants";
+import { createFinding, getAllLabels } from "./utils";
+
+const ethersProvider = getEthersProvider();
+const attackers = new Map<string, { origin: string; hops: number }>();
```

Comment:
Should `attackers` be persisted across different bot instances? If not, I suppose the only thing that wouldn't explicitly sync up between different bot instances is how the bot uses its own alerts (i.e. L76-79).

I feel it may not be an issue, but there could be something I'm overlooking.

Reply:
There is an edge case if some (but not all) nodes crash, for attackers last active before the `DAYS_TO_LOOK_BACK` (3 days) period who then execute a tx again, but we discussed with Carlos about avoiding storing to a db + I think that in V2 there'll only be one scanner per bot (?, I'm not really sure about that, I just think it makes sense given the RPC costs)